app/jobs/metais/sync_codelist_program_job.rb

```diff
@@ -0,0 +1,34 @@
+class Metais::SyncCodelistProgramJob < ApplicationJob
+  queue_as :metais
+
+  API_ENDPOINT = 'https://metais.vicepremier.gov.sk/cmdb/read/cilistfiltered'
+  REQUEST_BODY = '{"filter":{"type":["Program"],"metaAttributes":{"state":["DRAFT"]}},"page":1,"perpage":10000,"sortBy":"Gen_Profil_nazov","sortType":"ASC"}'
```

Comment:
Ak by si chcel mat istotu, ze tam nemas syntax chybu, tak sa to zvykne zapisat v ruby ako hash/array a zavolas potom na tom to_json, ale detail.